examples/foundational/14-function-calling.py

```diff
@@ -106,11 +111,11 @@ async def main():
 
         pipeline = Pipeline(
             [
-                fl_in,
+                # fl_in,
                 transport.input(),
                 context_aggregator.user(),
                 llm,
-                fl_out,
+                # fl_out,
```

Comment:
should we just remove the frame loggers?

Reply:
Probaby. I want to leave them in an example or two so people know they exist, though.

Reply:
If we leave them we should uncomment them I think.